tests/eval/test_password_sign_on.py

```diff
@@ -0,0 +1,257 @@
+"""
+This set of tests is based on the manual tests found in the
+IdP Release Candidate Test Matrix Version 4.2.4 (5/27/2021) here
+https://wiki.cac.washington.edu/display/SMW/IAM+Team+Wiki
+
+PWD-1 thru PWD-9.
+"""
+
+
+from functools import partial
+
+import pytest
+from webdriver_recorder.browser import Chrome
+
+from tests.helpers import CTRL_KEY, Locators, WebTestUtils
+from tests.models import ServiceProviderInstance, AccountNetid, TestSecrets
+
+
+@pytest.fixture
+def browser() -> Chrome:
+    return Chrome()
+
+
+@pytest.fixture(scope='class')
+def class_browser() -> Chrome:
+    browser = Chrome()
+    try:
+        yield browser
+    finally:  # Even if there is an error
+        browser.close()
+
+
+@pytest.fixture
+def new_tab(browser):
+    def inner():
+        browser.execute_script("window.open('');")
+        browser.switch_to.window(browser.window_handles[-1])
+    return inner
+
+
+@pytest.fixture
+def close_tab(browser):
+    def inner():
+        browser.find_element_by_tag_name('body').send_keys(CTRL_KEY + 'w')
+    return inner
+
+
+@pytest.fixture
+def netid() -> str:
+    return AccountNetid.sptest01.value
+
+
+@pytest.fixture
+def clean_browser(browser: Chrome) -> Chrome:
+    browser.delete_all_cookies()
+    return browser
+
+
+def add_suffix(suffix: str, netid: str) -> str:
+    return f'{netid}{suffix}'
+
+
+def crazy_casing(netid: str) -> str:
+    letters = list(netid)
+    for i, letter in enumerate(letters):
+        if i % 2 == 0:
+            letters[i] = letter.upper()
+    return ''.join(letters)
```

Comment:
Please remove once these are placed somewhere centrally, or migrated to the root `conftest.py` as fixtures